torch/_refs/__init__.py

```diff
@@ -879,6 +885,32 @@ def float_power(
     supports_lhs_python_scalar=False,
 )
 
+
+def _heaviside(input: TensorLikeType, values: TensorLikeType) -> TensorLikeType:
+    zeros_like_input = zeros_like(input)
```

Comment:
I don't think you need full size tensor here, 0 scalar should be enough, same for ones

Reply:
Hi @ngimel 

At this stage I cannot use `zeros_like_input = 0` and `ones_like_input = 1`.
The only working solution that I came up with is to use the smallest tensors of the same number of dimensions:
```
    zeros_like_input = zeros(shape=(1,)*input.ndim, dtype=input.dtype, device=input.device, requires_grad=False)
    ones_like_input = ones(shape=(1,)*input.ndim, dtype=input.dtype, device=input.device, requires_grad=False)
```
What do you think?

Reply:
Why can't use scalars?

Reply:
Example of refs.where accepting scalars: 

https://github.com/pytorch/pytorch/blob/bc1fef96aff4aeffe7a7b39ef8b4ff467860df28/torch/_refs/nn/functional/__init__.py#L312